### PR TITLE
Bugs fixes

### PR DESCRIPTION
**Pull request by pgilliar**
multplie bug fixes:
-wrong strings on qrcode popup windows
-no account follow
-default map location
-option menu translation
-zoom level bind between map and list

### CODE REVIEW

File: app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt [hunk 1]
```diff
@@ -148,8 +150,19 @@ class ProfileActivity : AppCompatActivity() {
         }
         followButton.setOnClickListener {
             if (followButton.text.equals(getString(R.string.follow))) {
-                viewModel.follow(loggedProfile.uuid, profileViewed.uuid)
-                followButton.text = getString(R.string.unfollow)
+                if(viewModel.profileService.getLoggedInUserID() == null){
+                    createPopUp(this,
+                        {
```
Comment (zaclegarssure) [inline app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt:155]: We now have a way to test the popups, but we can do it next week if needed.

**Review by zaclegarssure — APPROVED** [review]
Otherwise I do believe you for the maths since it seems to work, nice job.

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt [hunk 2]
```diff
@@ -405,11 +420,19 @@ class MapsFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickList
...
 
+
     private fun getZoomLevel(radius: Double?): Double {
-        val result = if (radius != null) {
-            val scale = radius / GMAP_PIXEL_RATIO
-            (16 - ln(scale)) / ln(2.0)
+        val result = if ( mapFragment.requireView().contentDescription == "MapReady" && radius != null){
+            val mapWidth = mapFragment.view?.width?.div(resources.displayMetrics.density)
+            val latitudinalAdjustment =
+                cos(Math.PI * map.cameraPosition.target.latitude / 180.0)
```
Comment (TheblueMan003) [inline app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt:428]: magic number 🧙‍♂️

File: app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt [hunk 3]
```diff
@@ -46,16 +47,17 @@ import com.maltaisn.icondialog.pack.IconPack
...
 
 @ExperimentalCoroutinesApi
 @AndroidEntryPoint
 class MapsFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickListener,
     SearchView.OnQueryTextListener {
 
-    private val BASE_ZOOM = 7.0
-    private val MAX_ZOOM = 12.0
-    private val GMAP_PIXEL_RATIO = 500
+    private val EQUATOR_LENGTH = 40075.004
```
Comment (TheblueMan003) [inline app/src/main/java/com/github/palFinderTeam/palfinder/map/MapsFragment.kt:59]: nitpeaking: could be in its own class

**Review by TheblueMan003 — APPROVED** [review]
Other than that looks fine

**Pull request merged**